src/main/java/com/severell/initializr/action/template/TemplateFileOperation.java

```diff
@@ -22,9 +22,12 @@
     private TemplateParameter parameter ;
     private String dirPath;
 
-    TemplateFileOperation(TemplateParameter param){
+    TemplateFileOperation(TemplateParameter param, String version){
         parameter = param;
-        dirPath = System.getProperty("java.io.tmpdir").concat( this.getFileSeparator()).concat(parameter.getName());
+        dirPath = Config.get("TEMPLATE_DIR", System.getProperty("java.io.tmpdir"))
```

Comment:
@Belovedbb I added this, so it will default to the tmp dir if TEMPLATE_DIR is not set. That way a dev doesn't have to manually configure it either

Reply:
Yeah, this is the ideal way to handle it!  I think the PR is okay then